src/lib.rs

```diff
@@ -58,6 +58,9 @@ pub mod arm;
 #[cfg(all(kernel_user_helpers, target_os = "linux", target_arch = "arm"))]
 pub mod arm_linux;
 
+#[cfg(any(target_arch = "riscv32", target_arch = "riscv64"))]
```

Comment:
NOTE: we'll be submitting a PR to rust-lang/rust that adds these two target architectures.